### PR TITLE
GM1.3 Doc updates

### PR DESCRIPTION
**Pull request by aheyne**
First batch of updated for GM1.3

Signed-off-by: Austin Heyne <aheyne@ccri.com>

### CODE REVIEW

File: geomesa-examples-authorizations/README.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ authenticate with GeoServer and LDAP to store authorizations
 
 Before you begin, you must have the following:
 
-* an instance of Accumulo 1.5 or 1.6 running on Hadoop 2.2.x
+* an instance of Accumulo 1.7 or 1.8 running on Hadoop 2.2.x
```
Comment (mdzimmerman) [inline geomesa-examples-authorizations/README.md:42]: Hmm, not sure what minimum Hadoop we should recommend.

File: geomesa-examples-authorizations/README.md [hunk 2]
```diff
@@ -191,6 +191,14 @@ user,admin
 
 > :warning: Note: A user cannot set authorizations unless the user has the System.ALTER_USER permission.
 
+Next we'll grant permissions to the read the appropriate tables to ``user``
```
Comment (mdzimmerman) [inline geomesa-examples-authorizations/README.md]: Is there an extra "the" in there?

**Review by mdzimmerman — APPROVED** [review]
Apart from some little things, looks good to me.

File: geomesa-examples-authorizations/README.md (hunk 2 above)
Comment (aheyne) [reply geomesa-examples-authorizations/README.md]: Yup, fixed

**Pull request merged**